### DISCUSSION TITLE
Converting fonts

### DISCUSSION THREAD

**Discussion by asherevan** [russhughes/st7789_mpy#171] [Q&A]
I have a TrueType font that I want to use with this driver. How would I do that?

**Comment by russhughes** [comment]
The font2bitmap.py program in the utils folder can create a Python module from a True Type font that the driver can use.

For example, this command will create a 32-pixel high font from the NonoSans-Regular.ttf True Type font.
```
./font2bitmap.py NotoSans-Regular.ttf 32 -c 0x20-0x7f >noto.py
```

**Comment by asherevan** [comment]
I tried it but it returned this error:

`Attribute error: ‘FreeTypeFont’ object has no attribute ‘get size’`
